src/Glpi/Application/ErrorHandler.php

```diff
@@ -391,7 +391,7 @@ public function handleException(\Throwable $exception, bool $quiet = false): voi
 
         $this->logErrorMessage($error_type, $error_description, $error_trace, $log_level);
         if (!$quiet) {
-            $this->outputDebugMessage($error_type, $error_description, $log_level);
+            $this->outputDebugMessage($error_type, \htmlspecialchars($error_description), $log_level);
```

Comment:
It will be escaped in `outputDebugMessage` directly.
```suggestion
            $this->outputDebugMessage($error_type, $error_description, $log_level);
```